### PR TITLE
Add timeout and conection timeout to logentries handler config definition

### PR DESCRIPTION
**Pull request by dcsg**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
You need to add the new options to the XSD file too.

File: DependencyInjection/MonologExtension.php [hunk 1]
```diff
@@ -572,6 +572,12 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['level'],
                 $handler['bubble'],
             ));
+            if (isset($handler['timeout'])) {
+                $definition->addMethodCall('setTimeout', [$handler['timeout']]);
```
Comment (xabbuh) [inline DependencyInjection/MonologExtension.php]: The short array notation is not compatible with PHP 5.3.

**Comment by dcsg** [comment]
@xabbuh they are not "new" since they are already defined for socket handler. But yes I don't see any timeout or connection_timeout in the .xsd file

**Comment by Seldaek** [comment]
Thanks, I added the missing stuff to xsd

**Pull request merged**